.travis.yml

```diff
@@ -3,7 +3,6 @@ sudo: false
 
 # Keep this in sync with appveyor.yml
 node_js:
-- '4'
 - '6'
```

Comment:
Can you add Node 8 to the list? That will help us prepare for the next [LTS release](https://github.com/nodejs/LTS). If adding that causes problems, we can do it in a separate issue but I don't foresee any issues.